docs/develop/json-manifest-overview.md

```diff
@@ -0,0 +1,871 @@
+
+> [!NOTE]
+> When it becomes possible to combine an add-in with other Microsoft 365 extension types in a single web app, then there will be other top-level properties that don't fit into the notion of the base manifest. Roughly speaking, there will be a top-level property for every kind of Microsoft 365 extension type, such as "configurableTabs", "bots" and "connectors". For examples, see the [Teams manifest documentation](/microsoftteams/platform/resources/schema/manifest-schema). This structure makes clear that the "extension" property represents an Office add-in as one type of Microsoft 365 extension.
+
+#### Base manifest
+
+The base manifest properties specify characteristics of the add-in that *any* type of extension of Microsoft 365, including Teams tabs or message extensions, would be expected to have, not just Office add-ins. These characteristics include a public name and a unique ID. The following table shows a mapping of some critical top level properties in the preview JSON manifest to the XML elements in the current manifest, where the mapping principle is the *purpose* of the markup.
+
+|JSON property|Purpose|XML element(s)|Comments|
+|:-----|:-----|:-----|:-----|
+|"$schema"| Identifies the manifest schema. | attributes of **OfficeApp** and **VersionOverrides** | |
+|"id"| GUID of the add-in. | **Id**| |
+|"version"| Version of the add-in. | **Version** | |
+|"manifestVersion"| Version of the manifest schema. |  attributes of **OfficeApp** | |
+|"name"| Public name of the add-in. | **DisplayName** | |
+|"description"| Public description of the add-in.  | **Description** | |
+|"accentColor"||| This property has no equivalent in the current XML manifest and is not used in the preview of the JSON manifest. But it must be present. |
+|"developer"| Identifies the developer of the add-in. | **ProviderName** | |
+|"localizationInfo"| Configures the default locale and other supported locales. | **DefaultLocale** and **Override** | |
+|"webApplicationInfo"| Identifies the add-in's web app as it is known in Azure Active Directory. | **WebApplicationInfo** | In the current XML manifest, the **WebApplicationInfo** element is inside **VersionOverrides**, not the base manifest. |
+|"authorization"| Identifies any Microsoft Graph permissions that the add-in needs. | **WebApplicationInfo** | See comment in preceding row. |
+
+The **Hosts**, **Requirements**, and **ExtendedOverrides** elements are part of the base manifest in the current XML manifest. But concepts and purposes associated with these elements are configured inside the "extension" property of the preview JSON manifest. 
```

Comment:
Does the concept of a "base manifest" exist in the XML version today?

Reply:
Yes. For an example, see https://docs.microsoft.com/en-us/office/dev/add-ins/develop/specify-office-hosts-and-api-requirements